pondo/pondo_core_protocol/src/main.leo

```diff
@@ -816,25 +1078,23 @@ program pondo_core_protocol.aleo {
     // At this point, all credits have been transferred to the core protocol, but there may still be commission owed
     let core_protocol_account: u64 = credits.aleo/account.get_or_use(self.address, 0u64);
     let reserved_for_withdrawal: u64 = balances.get(CLAIMABLE_WITHDRAWALS);
-    let deposit_pool: u64 = core_protocol_account - full_balance - reserved_for_withdrawal;
+    // Move bonded withdrawals to available to claim
+    balances.set(CLAIMABLE_WITHDRAWALS, reserved_for_withdrawal + bonded_withdrawals);
+    // Reset bonded withdrawals
+    balances.set(BONDED_WITHDRAWALS, 0u64);
+    // Calculate deposit pool
+    let deposit_pool: u64 = core_protocol_account - transfers_total - reserved_for_withdrawal;
 
     let new_commission_paleo: u64 = calculate_new_paleo(current_balance as u128, deposit_pool as u128, new_commission as u128, total_paleo_minted);
+    assert(commission_mint >= (current_owed_commission * 98u64) / 100u64); // Assert that the minted commission is at least 98% of the owed commission
```

Comment:
`Issue 03` We now assert that most of the owed commission is minted, but it is still possible for commission to sit if we miss a rebalance in a given epoch